components/ingest-service/backend/elastic/nodes.go

```diff
@@ -25,9 +25,17 @@ func (es *Backend) InsertNode(ctx context.Context, node backend.Node) error {
 	return err
 }
 
+// InsertUpsertNode inserts the UpsertNode with the created date
+func (es *Backend) InsertUpsertNode(ctx context.Context, node backend.UpsertNode) error {
+	mapping := mappings.NodeState
+	err := es.upsertDataWithID(ctx, mapping, node.EntityUuid, node)
+	return err
+}
+
 func (es *Backend) CreateBulkNodeUpdateRequest(node backend.Node) elastic.BulkableRequest {
 	mapping := mappings.NodeState
-	return es.createBulkRequestUpsertDataWithID(mapping, node.EntityUuid, node)
+	return es.createBulkRequestUpsertDataWithID(mapping, node.EntityUuid, node,
+		backend.UpsertNode{Node: node, Created: time.Now()})
```

Comment:
Create a Bulk Request with the UpserNode that contains the Created date. If the node is new it will use the UpsertNode to created. If it is an existing node it will use the node data to update it.